Objects/typevarobject.c

```diff
@@ -443,45 +443,39 @@ static PyMethodDef typevar_methods[] = {
 PyDoc_STRVAR(typevar_doc,
 "Type variable.\n\
 \n\
-Usage::\n\
+The preferred way to construct a type variable is via the dedicated syntax\n\
+for generic functions, classes, and type aliases:\n\
 \n\
-  T = TypeVar('T')  # Can be anything\n\
-  A = TypeVar('A', str, bytes)  # Must be str or bytes\n\
+    class Sequence[T]:  # T is a TypeVar\n\
+        ...\n\
 \n\
-Type variables exist primarily for the benefit of static type\n\
-checkers.  They serve as the parameters for generic types as well\n\
-as for generic function definitions.  See class Generic for more\n\
-information on generic types.  Generic functions work as follows:\n\
-\n\
-  def repeat(x: T, n: int) -> List[T]:\n\
-      '''Return a list containing n references to x.'''\n\
-      return [x]*n\n\
+This syntax can also be used to create bound and constrained type\n\
+variables:\n\
 \n\
-  def longest(x: A, y: A) -> A:\n\
-      '''Return the longest of two strings.'''\n\
-      return x if len(x) >= len(y) else y\n\
+    class StrSequence[S: str]:  # S is a TypeVar bound to str\n\
+        ...\n\
 \n\
-The latter example's signature is essentially the overloading\n\
-of (str, str) -> str and (bytes, bytes) -> bytes.  Also note\n\
-that if the arguments are instances of some subclass of str,\n\
-the return type is still plain str.\n\
 \n\
```

Comment:
Don't think two line breaks are necessary here

```suggestion
```

Reply:
Thanks, fixed.

Also thanks for all the thorough reviews! I am going to merge this now so I can stop worrying about conflicts.

Reply:
Yup, it's time :)